modules/dynamic-operator/api/v1alpha1/webhook_suite_test.go

```diff
@@ -190,3 +190,7 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).NotTo(HaveOccurred())
 })
+
+func useExistingCluster() bool {
```

Comment:
If you added function why you don't refactoring places where the function is not used?

Reply:
80 line at least

Reply:
fixed